integration-tests/ui-integration-tests/pom.xml

```diff
@@ -366,39 +383,54 @@
                         <goals>
                             <goal>exec</goal>
                         </goals>
-                        <phase>test</phase>
+                        <phase>integration-test</phase>
                         <configuration>
                             <executable>npm</executable>
                             <arguments>
                                 <argument>run</argument>
                                 <argument>test</argument>
-                                <argument></argument>
                                 <argument>skipTests=${maven.test.skip}</argument>
                             </arguments>
                         </configuration>
                     </execution>
-                    <execution>
-                        <id>npm run ui test (test)</id>
-                        <goals>
-                            <goal>exec</goal>
-                        </goals>
-                        <phase>test</phase>
-                        <configuration>
-                            <executable>npm</executable>
-                            <arguments>
-                                <argument>run</argument>
-                                <argument>ui-test</argument>
-                                <argument>--</argument>
-                                <argument>--skipTests=${maven.test.skip}</argument>
-                            </arguments>
-                        </configuration>
-                    </execution>
+                    <!--<execution>-->
```

Comment:
These will be removed after reasearch on testing framework for UI tests are done